packages/react-native-cli/src/commands/ConfigureCommand.ts

```diff
@@ -0,0 +1,30 @@
+import prompts from 'prompts'
+import logger from '../Logger'
+import onCancel from '../lib/OnCancel'
+import { addApiKey as addApiKeyAndroid } from '../lib/AndroidManifest'
+import { addApiKey as addApiKeyIos } from '../lib/InfoPlist'
+
+export default async function run (argv: string[], opts: Record<string, unknown>): Promise<void> {
+  const projectRoot = process.cwd()
+
+  try {
+    const { apiKey } = await prompts({
+      type: 'text',
+      name: 'apiKey',
+      message: 'What is your Bugsnag API key?',
+      validate: value => {
+        return value.length > 1
```

Comment:
Is there value in validating that the apiKey is a hexadecimal string of the correct length?